offchain/danaswap-ui/views/components/footer.ejs

```diff
@@ -0,0 +1,78 @@
+<footer class="footer p-6">
+  <div class="columns is-vcentered is-mobile">
+    <div class="column">
+      <a href="/">
+        <img
+          src="../../assets/images/logo_footer.png"
+          alt="Ardana"
+          title="Ardana"
+        />
+      </a>
+    </div>
+
+    <div class="column is-narrow">
+      <div class="columns is-vcentered is-mobile">
+        <div class="column">
+          <span class="icon">
```

Comment:
So we should still offer a title `<a title="x"><svg/><span class="srOnly">x</span></a>` for usability, accessibility, and SEO (changing the spans to as, would be better even if we don't have the href yet).

OR

`<a title="x"><svg><title>x</title><desc>x</desc><use/></svg></a>`

(both are equally accessible *and* show a title on hover)